src/block/WeightedPressurePlate.php

```diff
@@ -55,12 +55,7 @@ protected function calculatePlateState(array $entities) : array{
 		}
 		return [
 			(clone $this)->setOutputSignalStrength($newSignalStrength),
-			//I dare anyone to simplify this without breaking it
-			match(true){
-				$this->signalStrength === 0 && $newSignalStrength !== 0 => true,
-				$this->signalStrength !== 0 && $newSignalStrength === 0 => false,
-				default => null,
-			}
+			($value = $this->signalStrength === 0) === ($newSignalStrength !== 0) ? $value : null
```

Comment:
Are you sure it is really a good idea to make this even more complex?

Reply:
No, I'm not sure. But I may come up with an alternative.